paddle/fluid/operators/pool_mkldnn_op.cc

```diff
@@ -24,10 +24,13 @@ using mkldnn::pooling_backward;
 
 // Generate keys for storing/retriving primitives for this operator
 // TODO(jczaja): Make hashing function more optimial
-static std::string gethash(memory::dims& input_dims, std::string& pooling_type,
-                           std::vector<int>& ksize, std::vector<int>& strides,
-                           std::vector<int>& paddings, std::string suffix) {
-  auto dims2str = [](memory::dims& operand_dims) {
+static std::string gethash(const memory::dims& input_dims,
+                           const std::string& pooling_type,
+                           const std::vector<int>& ksize,
+                           const std::vector<int>& strides,
+                           const std::vector<int>& paddings,
+                           const std::string& suffix) {
```

Comment:
why do you modify the parameter order of `strides` and `paddings`? Could `pool_mkldnn_op.cc` remain as before?

Reply:
I modified these line of code, because the cpplint gives the message that I have to make these changes. Likely that is relate to the compiler version which is used by me.